### PR TITLE
initialize package

### PR DESCRIPTION
**Pull request by with-heart**
This PR initializes this project as a package. It also adds `husky`, `lint-staged`, and `prettier`, so we can have that sweet sweet `prettier` formatting on commit.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+
+.DS_Store
```
Comment (codingwithmanny) [inline .gitignore:3]:
```suggestion
.DS_Store
*.log
package-lock.json
```

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```
Comment (codingwithmanny) [inline .husky/pre-commit:4]:
```suggestion
./node_modules/.bin/line-staged
```

**Review by codingwithmanny — CHANGES_REQUESTED** [review]
Let me know what you think about the changes.

**Comment by Dhaiwat10** [comment]
Might be kinda off-topic so pardon me but I'd love to know the purpose behind making this a package 😄 Are gonna use this to serve data, or something else?

**Comment by with-heart** [comment]
> I think it's just to format the .md files.

Yeap that's literally it

**Comment by with-heart** [comment]
Probably some other things. But yeah that's it for now 😉

File: package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "scripts": {
+    "prepare": "husky install"
+  },
+  "devDependencies": {
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "prettier": "^2.4.1"
+  },
+  "lint-staged": {
+    "*.{json,md}": "prettier --write"
```
Comment (codingwithmanny) [inline package.json:15]:
```suggestion
    "*.{json,md}": "prettier --write *.md"
```

**Review by codingwithmanny — CHANGES_REQUESTED** [review]
Ignore...

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Pull request merged**